_includes/ballot-nav.html

```diff
@@ -1,8 +1,11 @@
 {% assign locality = include.locality %}
 {% assign tabIndxVar = 4 %}
 {% assign election_year = ballot.election | slice: 0, 4 %}
+{% assign next_year = election_year | plus:1 %}
```

Comment:
Let's have a comment here so that when we forget about this temporary fix until the next election cycle, we can understand why this was done.

Reply:
done